os_migrate/roles/prelude_common/templates/clouds.yaml.j2

```diff
@@ -0,0 +1,23 @@
+clouds:
```

Comment:
I think these parts were already merged, there may need to be a rebase of the PR to make this code disappear from the PR.

Reply:
All the auth replacement PRs should be merging into `feature/auth-replacement`. I don't recall what we needed to change in Makefile tho :)